astropy/utils/data.py

```diff
@@ -968,6 +979,18 @@ def download_file(remote_url, cache=False, show_progress=True, timeout=None):
         The timeout, in seconds.  Otherwise, use
         `astropy.utils.data.Conf.remote_timeout`.
 
+    sources : list of str, optional
+        If provided, a list of URLs to try to obtain the file from. The
+        result will be stored under the original URL. The original URL
+        will *not* be tried unless it is in this list; this is to prevent
+        long waits for a primary server that is known to be inaccessible
+        at the moment.
+
+    update_cache : bool, optional
```

Comment:
Yes, this is awkward. Compatibility poses a challenge too - existing code will call download_file with cache set to False or to the default to indicate that it wants the current state of the file checked (and maybe also to indicate that the file isn't worth caching?). But it would work to use "update" and True as inputs, or "update" and some other string to distinguish between checking for a new version or not.

Reply:
I like the idea of allowing special strings for `cache`.